modules/radix_sort/include/radix_sort.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Zoreev Mikhail
+
+#ifndef MODULES_RADIX_SORT_INCLUDE_RADIX_SORT_H_
+#define MODULES_RADIX_SORT_INCLUDE_RADIX_SORT_H_
+
+#include <algorithm>
+#include <limits>
+#include <random>
+#include <stdexcept>
+#include <vector>
+
+namespace RadixSort {
+    /**
+     * Sorts vector elements in range [begin, end)
+     * @param begin Iterator to begin element
+     * @param end Iterator to end element
+     */
+    template <typename ElementType>
+    void radixSort(typename std::vector<ElementType>::iterator begin,
+                   typename std::vector<ElementType>::iterator end) {
+        assert(begin < end);
+        assert(std::is_integral<ElementType>());
```

Comment:
* Выражение `std::is_integral<ElementType>::value` или `std::is_integral_v<ElementType>` более универсально и потому предпочтительнее, нежели `std::is_integral<ElementType>()`
* Эта информация известна в compile-time, поэтому здесь лучше использовать `static_assert`:
```suggestion
        static_assert(std::is_integral<ElementType>::value, "ElementType must be integral");
```
(или вовсе `std::enable_if`)

Reply:
Исправлено